images/ceph/Makefile

```diff
@@ -124,7 +124,7 @@ csv: $(OPERATOR_SDK) $(YQv3)
 csv-clean: ## Remove existing OLM files.
 	@rm -fr ../../build/csv/ceph/${go env GOARCH}
 	@rm -f ../../build/csv/rook-ceph.clusterserviceversion.yaml
-	@git restore $(MANIFESTS_DIR)/crds.yaml ../../deploy/charts/rook-ceph/templates/resources.yaml
+	@git restore $(MANIFESTS_DIR)/crds.yaml ../../deploy/charts/rook-ceph/templates/resources.yaml &
```

Comment:
Why do we need this restore in the first place? Does the build delete this file somewhere?

Reply:
Adding the `&` would turn it into a background process so I'm not seeing how this will help the issue of parallel builds.

Reply:
> Why do we need this restore in the first place? Does the build delete this file somewhere?

when we do make csv, it remove the description, so we need to revert the changes

Reply:
What if we instead cp the file to back it up before removing the descriptions, the mv it back? Then we don't need a git restore command. I still don't see how the background processes could solve the race condition.